.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## Pull Request Template for pycountry
+
+Thank you for contributing to pycountry! Before submitting this PR, please ensure you've completed the following steps:
+
+### Pre-Submission Checklist:
+- [ ] Code is properly formatted and adheres to the project's coding guidelines.
+- [ ] Changes have been tested with the latest version of the codebase.
+- [ ] Code changes do not decrease the current test coverage.
+- [ ] All existing tests pass successfully.
+- [ ] New tests (if applicable) are added to cover your changes.
+
+### Local Testing:
+Please run the following commands to ensure your changes do not introduce any new issues:
+- [ ] `pytest` - Runs all the tests to ensure they pass.
+- [ ] `make test` - Executes the custom test suite.
```

Comment:
`pytest` and `make test` shouldn't need to be run separately, `make test` runs `pytest` using every version we advertise support for (if available).

Reply:
@zware removed!